src/mono/mono/mini/aot-compiler.c

```diff
@@ -4371,10 +4331,14 @@ add_extra_method_full (MonoAotCompile *acfg, MonoMethod *method, gboolean prefer
 	}
 
 	if ((acfg->aot_opts.dedup || acfg->aot_opts.dedup_include) && mono_aot_can_dedup (method)) {
```

Comment:
There are flags that might be redundant:
 - `dedup` indicates dedup-skip phase
 - `dedup_include` indicates dedup-include phase
 -  `dedup_collect_only` indicates collecting of dedup-able methods in dedup-include
 - `dedup_emit_mode` indicates emitting of dedup-able methods in dedup-include phase

`dedup_collect_only` and `dedup_emit_mode` could be one flag that indicates collecting/emitting in dedup-include phase.